### ISSUE TITLE
Notebook Icons in the Contents/File explorer should reflect the default renderer

### ISSUE THREAD

**Issue by stevejpurves** [executablebooks/jupyterlab-mystjs#4]
### Context

When the mystjs extension is installed it becomes the default renderer for notebooks. 

Double clicking on a notebook in the content/file explorer opens the notebook with the MyST renderer -- however the notebook item displayed on items in the view do not show them as myst notebooks

![image](https://user-images.githubusercontent.com/1473646/212307126-55a86ab5-5f29-473b-95e7-d9e32b1cbc8f.png)

### Proposal

When the extension is installed, the file explorer icon should show the myst notebook icon, to make it clear which renderer will be used by default.

**Comment by stevejpurves** [comment]
@fcollonval do you know if Jupyterlab provides an API for extensions to do this?

**Comment by fcollonval** [comment]
@stevejpurves you can register a new file type; see e.g.

https://github.com/mwouts/jupytext/blob/aae7c64e12033bfe73e72536f068932f3d6ac06f/packages/labextension/src/index.ts#L427

Unfortunately there is no ranking for file type. So the icon will be the one of the first loaded file type for the notebook file - I think the extension won't take over.

Alternatively, you could dynamically change the icon SVG for notebook. You could do (I never tried that):

```typescript
import { notebookIcon } from '@jupyterlab/ui-components';

// When the extension get activated
notebookIcon.svgstr = '...'; // Set new svg for the notebook icon
```

**Comment by agoose77** [comment]
Think this is fixed by #16, in the sense that we are now _the_ notebook renderer. Let me know if this was premature.
